fhd_core/setup_metadata/load_skyh5_source_catalog.pro

```diff
@@ -0,0 +1,33 @@
+FUNCTION load_skyh5_source_catalog, catalog
+  ; Restore a source catalog from a .skyh5 file
+  
+  file_id = H5F_OPEN(catalog)
+  dataset_id = H5D_OPEN(file_id, '/Data/stokes')
+  stokes = H5D_READ(dataset_id)
+
+  nsources = H5D_READ(H5D_OPEN(file_id, '/Header/Ncomponents'))
```

Comment:
I would suggest making each of these names (e.g. `'/Header/Ncomponents'`) keyword arguments with these values as defaults. Ignore this suggestion if these names must never be different than the values here.